src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -291,14 +291,14 @@ class AutoRerouteSpec extends HealthCheckSpecification {
         allFlowPaths.findAll { it != flowPath }.each { pathHelper.makePathMorePreferable(it, flowPath) }
 
         when: "Bring the flow port down on the source switch"
-        antiflap.portDown(flow.source.datapath, flow.source.portNumber)
+        northbound.portDown(flow.source.datapath, flow.source.portNumber)
 
         then: "The flow is not rerouted"
         TimeUnit.SECONDS.sleep(rerouteDelay)
         PathHelper.convert(northbound.getFlowPath(flow.id)) == flowPath
 
         when: "Bring the flow port down on the destination switch"
-        antiflap.portDown(flow.destination.datapath, flow.destination.portNumber)
+        northbound.portDown(flow.destination.datapath, flow.destination.portNumber)
```

Comment:
you can't do northbound port down here and do antiflap port up in cleanup. Makes no sense. Also, why this change happen at the first place?